src/frontend/src/views/Data/cellRenderers.tsx

```diff
@@ -11,18 +11,32 @@ import {
   createTreeModalInfo,
   stringGuard,
 } from "src/common/utils";
+import { GISAIDCell, Subtext } from "./style";
+import { GISAID_STATUS_TO_TEXT } from "./utils/samples";
 
 const DEFAULT_RENDERER = (value: JSONPrimitive): JSX.Element => {
   return <div className={dataTableStyle.cell}>{value}</div>;
 };
 
 const SAMPLE_CUSTOM_RENDERERS: Record<string | number, CellRenderer> = {
-  privateId: (value: JSONPrimitive): JSX.Element => (
-    <div className={dataTableStyle.cell}>
-      {<SampleIcon className={dataTableStyle.icon} />}
-      {value}
-    </div>
-  ),
+  gisaid(value): JSX.Element {
```

Comment:
Add custom renderer for `gisaid` cell

Reply:
In a previous iteration of this I added a reference to the parent object the value came from in case we needed to reference more than one value in a column, I think that would be a better approach.

Reply:
Ah yes! In my Pangolin PR, renderer will accept an object as an argument `gisaid({ value, item, index })`, so that will be fixed there. Currently the `item` is the second argument of this render function. E.g, `gisaid(value, item, index)`